### PR TITLE
xds service discovery performance test document

### LINKED ISSUE #247: Update the benchmark test document
### What would you like to be added?

Add Performance Test Result

### Why is this needed?

Update the benchmark test document

### PR DESCRIPTION
**Pull request by daizhenyu**
What type of PR is this?

/Doc

What this PR does / why we need it?

xds service discovery performance test document

Which issue(s) this PR fixes:

Fixes #247

### CODE REVIEW

File: docs/zh/document/faq/performance.md [hunk 1]
```diff
@@ -221,4 +221,79 @@ Sermant版本：[`Release v1.4.0`](https://github.com/sermant-io/Sermant/release
...
+#### 测试结果
+
+| 测试场景             | 内存增加总量 | 堆内存增加 | 堆外内存增加 | class增加量 | CPU占用率增加 |
+| -------------------- | ------------ | ---------- | ------------ | ----------- | ------------- |
+| 挂载Sermant不开启xDS | 13.45M       | 0.45M      | 13M          | 1902        | 0%            |
+| 挂载Sermant开启xDS   | 35.37M       | 2.37M      | 33M          | 3837        | 0%            |
+
+#### 总结
+
+1. 宿主服务只挂载Sermant，不开启xDS服务时，Sermant框架共增加13.45M内存。其中，对外内存中增加1902个class对象
```
Comment (lilai23) [inline docs/zh/document/faq/performance.md]: 堆外

File: docs/zh/document/faq/performance.md [hunk 2]
```diff
@@ -221,4 +221,79 @@ Sermant版本：[`Release v1.4.0`](https://github.com/sermant-io/Sermant/release
...
+本次测试使用华为云容器引擎CCE进行应用部署，K8s集群的ECS节点数量为2个，规格如下：
+
+```
+规格：通用计算型｜8vCPUs｜24GiB
+Docker Version: v18.09.0
+Kubernetes Version: v1.21.7
+Istio Version：v1.14.3
+```
+
+K8s中所有应用Pod的规格一致，均为`2vCPUs|4GiB`。
```
Comment (lilai23) [inline docs/zh/document/faq/performance.md]: add envoy resource

File: docs/zh/document/faq/performance.md [hunk 3]
```diff
@@ -221,4 +221,80 @@ Sermant版本：[`Release v1.4.0`](https://github.com/sermant-io/Sermant/release
...
+
+| 测试场景             | 内存增加总量 | 堆内存增加 | 堆外内存增加 | class增加量 | CPU占用率增加 |
+| -------------------- | ------------ | ---------- | ------------ | ----------- | ------------- |
+| 挂载Sermant不开启xDS | 13.45M       | 0.45M      | 13M          | 1902        | 0%            |
+| 挂载Sermant开启xDS   | 35.37M       | 2.37M      | 33M          | 3837        | 0%            |
+
+#### 总结
+
+1. 宿主服务只挂载Sermant，不开启xDS服务时，Sermant框架共增加13.45M内存。其中，对外内存中增加1902个class对象
+2. 宿主服务挂载Sermant开启xDS服务增加的堆外内存属于正常范围，堆外内存共增加33M，相较于仅挂载Sermant堆外内存增加20M，主要为堆外内存新增1935个class对象，和xDS依赖和grpc依赖相关。同时，我们和主流的无代理服务治理框架进行了参考对比，开启xDS服务时增加的堆外内存是正常现象。
```
Comment (hanbingleixue) [inline docs/zh/document/faq/performance.md]: 增加的堆外内存在正常范围

File: docs/zh/document/faq/performance.md (hunk 1 above)
Comment (daizhenyu) [reply docs/zh/document/faq/performance.md]: done

File: docs/zh/document/faq/performance.md (hunk 2 above)
Comment (daizhenyu) [reply docs/zh/document/faq/performance.md]: done

File: docs/zh/document/faq/performance.md (hunk 3 above)
Comment (daizhenyu) [reply docs/zh/document/faq/performance.md]: done

**Review by lilai23 — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Review by lilai23 — APPROVED** [review]

**Pull request merged**